assembl/templates/base.jinja2

```diff
@@ -6,20 +6,46 @@
     <title>{% block page_title %}Assembl{% endblock %}</title>
     <meta name="viewport" content="width=device-width, initial-scale = 1,user-scalable=no,maximum-scale=1.0">
 
-    {% block meta_tags %}
-    <meta name="description" content="">
-    <meta property="og:title" content="Assembl" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="{{ application_url }}" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/crowd2.jpg" />
-    <meta property="og:image:width" content="1600" />
-    <meta property="og:image:height" content="222" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/logo.png" />
-    <meta property="og:image:width" content="111" />
-    <meta property="og:image:height" content="35" />
-    <meta name="robots" content="noindex" />
-    <meta name="googlebot" content="noindex" />
-    {% endblock meta_tags %}
+    {% block meta_tags_public %}
```

Comment:
You should probably also do the same in `templates/index_react.jinja2` than in `templates/index.jinja2`: have the same `private_social_sharing` condition and the different cases of general debate (landing page and any non-idea and non-post URL), idea page, and direct link to a message. Or maybe the scope of this task was only about the landing page? (I think while we are at it, it is good to do it for ideas and messages)

Reply:
It is done, however, I believe that later on I should probably modify the index file ... @ayyazdaniaryan  said that it is not recommended to do function calls in jinja2 templates, so probably later I will modify this file again to only use variables and not function calls